OverlayPlugin.Core/MemoryProcessors/Combatant/CombatantMemory70.cs

```diff
@@ -162,6 +160,8 @@ private unsafe struct CombatantMemory
             [FieldOffset(0xD0)]
             public Single Radius;
 
+            // Fields after 0x110 are only for PCs, Monsters, NPCs (mainly empty), and Retainers.
+
```

Comment:
(also move this comment down a bit to reflect the correct address)

```suggestion
            // All combatants have the first 0x1A0 bytes (`GameObject` per FFXIVClientStructs)
            // Fields below this point are only valid for PCs, Monsters, NPCs (mainly empty), and Retainers.

```